CHANGELOG.md

```diff
@@ -1,5 +1,96 @@
 # BEM-XJST Changelog
 
+# 2016-01-29, [v5.0.0](https://github.com/bem/bem-xjst/compare/v4.3.3...v5.0.0), @miripiruni
+
+Breaking changes: BEM-XJST now supports two template engines: BEMHTML for getting HTML output and BEMTREE for getting BEMJSON. By default BEM-XJST will use BEMHTML engine.
```

Comment:
As almost nobody use `bem-xjst` without `enb-bemxjst` consider moving this change under mods VS elemMods changes which will affect everybody.